packages/block-editor/src/components/use-editor-fonts-resolver/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useMemo, useCallback } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { getDisplaySrcFromFontFace, loadFontFaceInBrowser } from './utils';
+import { store as editorStore } from '../../store';
```

Comment:
Let's use `blockEditorStore` to be specific and differentiate from the editor package's store.

```suggestion
import { store as blockEditorStore } from '../../store';
```

Reply:
This part of the code was removed. Marking this comment as resolved.